packetery/src/Packetery/Carrier/class-repository.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Packeta carrier repository
+ *
+ * @package Packetery
+ */
+
+declare( strict_types=1 );
+
+namespace Packetery\Carrier;
+
+/**
+ * Class CarrierRepository
+ *
+ * @package Packetery
+ */
+class Repository {
+
+	/**
+	 * Gets wpdb object from global variable and sets custom tablename.
+	 *
+	 * @return \wpdb
+	 */
+	private static function get_wpdb(): \wpdb {
+		global $wpdb;
```

Comment:
```
$container = $configurator->createContainer();
$container->addService('wpdb', $wpdb);
```

a pak `__construct(\wpdb $wpdb) { ... }`

Reply:
A jak by se to potom použilo, aby to vzal sniffer?

Reply:
Je lepší udělat tovarničku a zaregistrovat jí v NEONu. (viz Latte engine factory)

Reply:
"Use placeholders and $wpdb->prepare(); found $this"